Mathlib/AlgebraicGeometry/EllipticCurve/Weierstrass.lean

```diff
@@ -85,25 +85,20 @@ universe s u v w
 /-- A Weierstrass curve $Y^2 + a_1XY + a_3Y = X^3 + a_2X^2 + a_4X + a_6$ with parameters $a_i$. -/
 @[ext]
 structure WeierstrassCurve (R : Type u) where
-  (a₁ a₂ a₃ a₄ a₆ : R)
+  /-- The `a₁` coefficient of a Weierstrass curve. -/
+  a₁ : R
+  /-- The `a₂` coefficient of a Weierstrass curve. -/
+  a₂ : R
+  /-- The `a₃` coefficient of a Weierstrass curve. -/
+  a₃ : R
+  /-- The `a₄` coefficient of a Weierstrass curve. -/
+  a₄ : R
+  /-- The `a₆` coefficient of a Weierstrass curve. -/
+  a₆ : R
```

Comment:
These are obviously pretty useless docstrings, but I just moved them from a few lines below!

Reply:
You can use a common docstring for all coefficients. Something like
```lean
  /-- Coefficients of a Weierstrass curve (write eqn). -/
  (a₁ a₂ a₃ a₄ a₆ : R)
```

Reply:
This will result in the same docstring being repeated 6 times in the docs, which seems worse than these docstrings. I'll push an alternate change.